plugins/CheckPrescale.cc

```diff
@@ -32,7 +32,8 @@ class CheckPrescale : public edm::EDAnalyzer {
     int hlt;
   };
 
-  tree_t last_t, t;
+  //tree_t last_t, t;
```

Comment:
Is last_t no longer used?

Reply:
As far as I can tell from the commit history of CheckPrescale.cc last_t was used in 2011. That is, the part of the code where it is actually used was commented out in 2011. All I did was comment out some of the remaining vestigial lines of code.